### PR TITLE
Bump to version 0.18

### PR DESCRIPTION
**Pull request by ZhukovGreen**
Changes in this version include:
- Prevent conversion failure by requiring writable /sys and /mnt
- Fix the subscription-manager packages being signed by CentOS instead of Red Hat
- Remove packages causing dependency errors
- Inhibit the conversion for UEFI firmware
- Added --version option
- Implement a backup of repofiles under /etc/yum.repos.d/
- Fix conversion failure on CentOS Linux/OL 6 caused by java-1.7.0-openjdk
- Remove settings not compatible with RHEL from /etc/sysconfig/kernel

### CODE REVIEW

**Review by fellipeh — APPROVED** [review]
LGTM when all green

File: packaging/convert2rhel.spec [hunk 1]
```diff
@@ -107,6 +107,13 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Mar 22 2021 Michal Bocek <mbocek@redhat.com> 0.18
+- Add check for read only sys and mnt
+- Reinstall subscription manager (for oracle linux)
+- Add abrt-cli package to excluded packages
```
Comment (bocekm) [inline packaging/convert2rhel.spec:113]: Replace with `Remove packages causing dependency errors`

**Review by bocekm — CHANGES_REQUESTED** [review]
the changelog should also include:
- Add --version option (https://github.com/oamg/convert2rhel/pull/176)
- Implement a backup of repofiles under /etc/yum.repos.d/ (https://github.com/oamg/convert2rhel/pull/127)
- Fix conversion failure on CentOS Linux/OL 6 caused by java-1.7.0-openjdk (https://github.com/oamg/convert2rhel/pull/167)
- Remove settings not compatible with RHEL from /etc/sysconfig/kernel (https://github.com/oamg/convert2rhel/pull/168)

File: packaging/convert2rhel.spec [hunk 2]
```diff
@@ -107,6 +107,13 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
...
+- Inhibit the convertion for UEFI firmware
```
Comment (bocekm) [inline packaging/convert2rhel.spec:114]:
```suggestion
- Inhibit the conversion for UEFI firmware
```

File: packaging/convert2rhel.spec [hunk 3]
```diff
@@ -107,6 +107,13 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
...
+- Inhibit the conversion when 3rd party or not existing kmods found on system
```
Comment (bocekm) [inline packaging/convert2rhel.spec:115]:
```suggestion
```
Remove this one as it's not been merged yet.

File: packaging/convert2rhel.spec [hunk 4]
```diff
@@ -107,6 +107,13 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Mar 22 2021 Michal Bocek <mbocek@redhat.com> 0.18
+- Add check for read only sys and mnt
```
Comment (bocekm) [inline packaging/convert2rhel.spec:111]:
```suggestion
- Prevent conversion failure by requiring writable /sys and /mnt
```

File: packaging/convert2rhel.spec [hunk 5]
```diff
@@ -107,6 +107,13 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
...
+- Reinstall subscription manager (for oracle linux)
```
Comment (bocekm) [inline packaging/convert2rhel.spec:112]:
```suggestion
- Ensure the subscription-manager on OL 6 is signed by Red Hat
```

**Review by bocekm — CHANGES_REQUESTED** [review]

**Review by bocekm — APPROVED** [review]

File: packaging/convert2rhel.spec [hunk 6]
```diff
@@ -107,6 +107,16 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Mar 22 2021 Michal Bocek <mbocek@redhat.com> 0.18
```
Comment (bocekm) [inline packaging/convert2rhel.spec:110]:
```suggestion
* Mon Mar 22 2021 Michal Bocek <mbocek@redhat.com> 0.18-1
```

**Review by bocekm — CHANGES_REQUESTED** [review]

**Review by bocekm — APPROVED** [review]

**Pull request merged**